src/app/core/record-works/record-works.service.ts

```diff
@@ -165,6 +165,12 @@ export class RecordWorksService {
    * @param options
    */
   getFeaturedWorks(options: UserRecordOptions): Observable<WorksEndpoint> {
+    options.pageSize = options?.pageSize || DEFAULT_PAGE_SIZE
```

Comment:
Seems like the `options` set here on the level of the class would overwrite the options set by the `getWorks`. 

Which I think could interfier if a user tries for instance to `updateVisibility` functionality